### PR TITLE
Sqlite and databases finally working 

### PR DESCRIPTION
**Pull request by Thyme-Cat**
There is room for improvement in the DBFacade.cs file, but according to the KISS principle it is good enough.
There is now made a chirp.db database file, and added the text from schema.sql and dump.sql correctly. 
Some guidance was given by group 3.

Co-authored-by: Jasmin <jsho@itu.dk>
Co-authored-by: Maria <arnv@itu.dk>
Co-authored-by: Henriette <hoes@itu.dk>
Co-authored-by: Sophie Gabriela <slak@itu.dk>
Co-authored-by: group 3

### CODE REVIEW

**Comment by JaSNOHo** [comment]
lgtm

File: CheepService.cs [hunk 1]
```diff
@@ -1,3 +1,8 @@
+using System.Data;
+using Microsoft.Data.Sqlite;
+//using Chirp.Database;
```
Comment (tomat-suppe) [inline CheepService.cs:3]: should we not remove this and below out-commented, old code?

File: CheepService.cs [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+//namespace Chirp.Cheeps;
```
Comment (tomat-suppe) [inline CheepService.cs:5]: remove?

File: DBFacade.cs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+public interface IDBFacade
+{
+    public List<CheepViewModel> DatabaseConnection();
+}
+
+public class DBFacade : IDBFacade
+{
+    private string SqlDBFilePath = "data/chirp.db";
+    private Boolean cheepdataExists = false;
+    //private string sqlQuery = "";
```
Comment (tomat-suppe) [inline DBFacade.cs:19]: is this still relevant?

File: DBFacade.cs [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    private static string FromAuthorIdToUserName(string author)
+    {
+        using (var connection = new SqliteConnection("Data Source=/tmp/chirp.db"))
+        {
+            connection.Open();
+
+            var sqlQuery = "SELECT user.*, message.* FROM user, message WHERE user.user_id = " + author;
+            //var username = "SELECT user_id FROM user JOIN message ON user.user_id = message.author_id WHERE user_id = " + author;
```
Comment (tomat-suppe) [inline DBFacade.cs:97]: is this and below still needed?

File: DBFacade.cs [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                var name = reader1.GetString(1);
+                return name;
+            }
+        }
+        return "";
+    }
+
+
+
+    //command.CommandText = sqlQuery;
```
Comment (tomat-suppe) [inline DBFacade.cs:116]: reason for keeping this comment?

**Review by tomat-suppe — COMMENTED** [review]
Have added some comments upon out-commented code that we should maybe just remove? Otherwise, functional code lgtm!! <3 Queen behavior!!!

File: DBFacade.cs (hunk 5 above)
Comment (Thyme-Cat) [reply DBFacade.cs:116]: It shall be removed.

File: DBFacade.cs (hunk 4 above)
Comment (tomat-suppe) [reply DBFacade.cs:96]: lets keep it queen

**Review by tomat-suppe — APPROVED** [review]
lgtm, let's push it to main

**Review by llsa-itu — APPROVED** [review]

**Review by llsa-itu — APPROVED** [review]

**Review by JaSNOHo — APPROVED** [review]
lgtm

**Pull request merged**